trainingMaquetado/index.html

```diff
@@ -13,6 +14,27 @@
       <img src="photos/tcg_logo.png">
     </div>
     <div class="middle-container">
+      <div class="item-container">
+        <div>
```

Comment:
you don't need to wrap the image inside a div. Always try to avoid unnecessary wrapper elements